pages/platform/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Platform"
+---
```

Comment:
This would still appear on search, and thus users can just search Platform and still land on the welcome page from our search bar.